cfgmgr/buffermgr.cpp

```diff
@@ -244,10 +271,78 @@ task_process_status BufferMgr::doSpeedUpdateTask(string port, bool admin_up, con
 
         fvVectorPg.push_back(make_pair("profile", profile_ref));
         m_cfgBufferPgTable.set(buffer_pg_key, fvVectorPg);
+        // Save the applied lossless PG
+        m_portLosslessPgs[port] = pfc_enable;
+        // Save the profile_ref
+        m_portPgLookup[port] = profile_ref;
     }
     return task_process_status::task_success;
 }
 
+void BufferMgr::updatePortLosslessPg(const string &port)
```

Comment:
remove. we don't need to update lossless pg's on the fly. Many asics do not support the buffer pg deletion. Hence you don't need to record them

Reply:
Removed.